packages/components/src/components/domainproperties/issues/actions.ts

```diff
@@ -0,0 +1,54 @@
+import {IssuesListDefModel} from "./models";
+import {ActionURL, Ajax, Domain, getServerContext, Utils} from "@labkey/api";
+
+export function fetchIssuesListDefDesign(issueDefName: string): Promise<IssuesListDefModel> {
+    return new Promise((resolve, reject) => {
+
+        Domain.getDomainDetails({
+            containerPath: getServerContext().container.path,
+            schemaName: 'issues',
+            queryName: issueDefName,
+            success: data => {
+                resolve(IssuesListDefModel.create(data));
+            },
+            failure: error => {
+                reject(error);
+            },
+        });
+    })
+
+}
+
+export function getUsersForGroup(groupId: any): Promise<any> {
```

Comment:
I believe the type of the `groupId` param should be `number` instead of `any`, right?

Reply:
Right, Now changed!